### PR TITLE
Remove VERSION and add version.py

### PR DESCRIPTION
**Pull request by silverv**
This is necessary pycyphal to be installable using pyinstaller.

### CODE REVIEW

File: pycyphal/__init__.py [hunk 1]
```diff
@@ -32,10 +32,10 @@
 """
 
 import os as _os
-from importlib.resources import read_text as _read_text
 
 
-__version__ = _read_text(__name__, "VERSION", encoding="utf8").strip()
+from version import __version__ as _v
+__version__ = _v
```
Comment (pavel-kirienko) [inline pycyphal/__init__.py:38]: Can't you just say `from version import __version__ as __version__`?

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]
I think `version.py` should be `_version.py` because it is not part of the API.

File: setup.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name            = pycyphal
-version         = file: pycyphal/VERSION
+version         = attr: pycyphal.__version__
```
Comment (pavel-kirienko) [inline setup.cfg:3]: This import will fail if the library dependencies (like NumPy) are not installed. Did you mean to say `pycyphal._version.__version__`?

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]

**Comment by silverv** [comment]
As discussed we should look at the CI green of the previous commit and consider the changes made inbetween trivial. Maybe running tests locally to save 4 hours.

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]
Please run black (the code is misformatted) and leave CI running (do not cancel it).

**Comment by silverv** [comment]
It's not misformatted and I will not cancel the CI anymore

**Review by pavel-kirienko — APPROVED** [review]

**Pull request merged**